src/splatting/Splatting.js

```diff
@@ -599,27 +617,85 @@ void main () {
     vPosition = position;
 
     vec2 vCenter = vec2(pos2d) / pos2d.w;
+    
+    
+    // implement accurate mouse raycasting
+    float majorAxisOffset = dot(uMouse - vCenter, normalize(majorAxis)) / ( abs(position.x) * length(majorAxis / viewport) );
+    float minorAxisOffset = dot(uMouse - vCenter, normalize(minorAxis)) / ( abs(position.y) * length(minorAxis / viewport) );
+    vec4 actualCam = cam;
+    vec4 offset1 = vec4(majorAxisOffset * (abs(position.x) * majorAxis / viewport), 0., 0.);
+    offset1 = inverse(projection) * (offset1 * pos2d.w);
+    vec4 offset2 = vec4(minorAxisOffset * (abs(position.y) * minorAxis / viewport), 0., 0.);
+    offset2 = inverse(projection) * (offset2 * pos2d.w);
+    actualCam = actualCam + offset1 + offset2;
+    vFBOPosColor = inverse(view) * actualCam;
+    vFBOPosColor.a = 1.;
+    
+    if (vFBOPosColor.xyz == uCollidePosition) { // todo Steve new 11: directly compare the position here and see if it works
```

Comment:
Has an issue been created for this todo? Otherwise this will be forgotten and if it works it might be better to remove the comment.

Reply:
Oh I forgot to delete it